### PR TITLE
added media query for bg color only for desktop view

### PR DESCRIPTION
**Pull request by Smit2808**

### CODE REVIEW

File: mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss [hunk 1]
```diff
@@ -232,9 +232,10 @@
 
 	See: https://github.com/WordPress/wporg-mu-plugins/issues/159
 	*/
-
-	& .wp-block-navigation__container {
-		background: var(--wp--preset--color--dark-grey);
+	@media (--tablet) {
+		& .wp-block-navigation__container {
+			background: var(--wp--preset--color--dark-grey);
+		}
```
Comment (iandunn) [inline mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss:239]: I like to put the media query inside the element. That way default mobile styles can go in there too without having to create a duplicate block. eg.,

```postcss
& .wp-block-navigation__container {
	/* other stuff can go here in the future */

	@media (--tablet) {
		background: var(--wp--preset--color--dark-grey);
	}
}
```

**Comment by iandunn** [comment]
fixes #196

**Comment by Smit2808** [comment]
Hi @iandunn

Thank you for reviewing my PR. As you have added the "fixes" in the latest comment, So now do I need to change anything as you said in the first comment to add media query inside the element?

**Comment by iandunn** [comment]
Ah, no, that was just to [link the PR to the issue](https://github.blog/2013-01-22-closing-issues-via-commit-messages/).

I think what's remaining here is just to move the media query inside the element, and then hopefully @StevenDufresne can test in Falkon since he's already got a VM setup for that.

**Comment by StevenDufresne** [comment]
I have preemptively tested the changes that @iandunn is requesting. Works as expected 👍. If you can update the PR, we can merge it in!

**Pull request closed without merging**